src/main/java/de/tum/cit/aet/artemis/quiz/domain/QuizQuestionProgress.java

```diff
@@ -0,0 +1,73 @@
+package de.tum.cit.aet.artemis.quiz.domain;
+
+import java.time.ZonedDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+
+import de.tum.cit.aet.artemis.core.domain.User;
+
+@Entity
+public class QuizQuestionProgress {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @ManyToOne
+    @JoinColumn(name = "quiz_question_id")
+    private QuizQuestion quizQuestion;
+
+    @Column(name = "progress_json", columnDefinition = "json")
+    private String progressJson;
```

Comment:
With this your Java could would be unaware of how the data is stored, as Hibernate automatically converts to/from JSON
```suggestion
    @JdbcTypeCode(SqlTypes.JSON)
    @Column(name = "progress_json", columnDefinition = "json")
    private QuizQuestionProgressData progress;
```